site/_posts/2020-08-11-advanced-registration-form.md

```diff
@@ -0,0 +1,176 @@
+
+<!--more-->
+
+FusionAuth is an auth system, but it also provides user management, registration and more. With FusionAuth, you can build custom registration forms to capture the information you need. In this blog post, you'll build a self service registration form for a (fake) real estate application. When the user registers, they'll see two steps, one for basic account information and the other to capture additional information about their home buying needs. 
+
+If you follow this tutorial, you'll end up with a form that asks a new user for the following:
+
+* Email
+* Password
+* First name
+* Phone number
+* Geographic area where they are looking
+* Minimum house price
+* Maximum house price
+
+Because this is a lot of information to ask for at once, you'll break this up into two pages. We'll be walking through this using the administrative user interface, but everything can be done via [the APIs](/docs/v1/tech/apis/forms), should you need to manage registration forms programmatically.
+
+## FusionAuth setup
+
+If you don't have FusionAuth running, [get it going in 5 minutes](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide). 
+
+Then get a license key and activate it. *Please note that advanced registration forms are a paid edition feature. You can [learn more about paid editions and sign up for a free trial here](/pricing).* 
+
+Next, [activate your license](/docs/v1/tech/reactor). Whew, all the FusionAuth set up is done. 
+
+## Create form fields
+
+Now you need to create custom form fields. To do so, navigate to "Customizations" and then "Form fields" after logging into the administrative user interface. You'll see some form fields are already there. These are the default fields available for your registration forms: 
+
+{% include _image.liquid src="/assets/img/blogs/advanced-registration-forms/predefined-keys.png" alt="A list of the predefined form fields that you can use for registration." class="img-fluid" figure=false %}
+
+You can mix and match these form fields on a new form. If that's all you need, you can avoid creating custom form fields. But if you need anything new, you'll want to create a new form field.
+
+### Custom form fields
+
+The real power of registration form building can be seen when you add custom fields. You can add as many of these as you'd like. 
+
+You can store values in any of the predefined user fields such as `user.fullName`. But you can also use the `data` field on both the `registration` and the `user` objects to store arbitrary data. 
+
+`registration.data` is the right place to store data related to a user's account but specific to an application. As a reminder, [a registration](/docs/v1/tech/core-concepts/registrations) is a link between a user and an application. 
```

Comment:
I'd reword: "`registration.data` is the right place to store data related to a user's account but specific to an application" to "`registration.data` is the right place to store data related to a user's account that is specific to an application"

Reply:
Sounds good!